draft-ietf-quic-multipath.md

```diff
@@ -1197,12 +1197,22 @@ Path Identifier:
 Note that, other than for the NEW_CONNECTION_ID frame of {{Section 19.15 of QUIC-TRANSPORT}},
 the sequence number applies on a per-path context.
 This means different connection IDs on different paths may have the same
-sequence number value. Respectively, the Retire Prior To field indicates which connection IDs
-should be retired for the path with the Path ID in the Path Identifier field.
+sequence number value. However, the value of the Connection ID MUST be unique
+across all paths.
+
+The Retire Prior To field indicates which connection IDs
+should be retired among those that share the Path ID in the Path Identifier field.
+Connection IDs associated with different path IDs are not affected.
 
 Note that the NEW_CONNECTION_ID frame can only be used to issue or retire
 connection IDs for the initial path with Path ID 0.
 
+Endpoints SHOULD NOT issue updates of the Retire Prior To field for a specific path ID
```

Comment:
Let's not use normative language here is it is already specified normatively in RFC9000.

Reply:
We can have more than one sentence but I don't think we should normatively rephrase things from RFC9000. So if you add text that ensure it is well understood what we mean without using normative language that would be my preferred option.

Reply:
OK, updating the text.